cmd/minishift/cmd/start.go

```diff
@@ -677,6 +681,30 @@ func initSubscriptionManagerFlags() *flag.FlagSet {
 	return subscriptionManagerFlagSet
 }
 
+// Get config from the start flags and set it to viper config so that in
+// stop start case user don't need to remember the start flags.
+// Note: This is only going to work for start flag not cluster up flags.
```

Comment:
can you explain? Clusterup flags in that case also need to be persisted and re-applied on start.

Reply:
@gbraad as of now we are only taking care of `startFlagSet` not the `clusterupFlagSet` because with stop and start we don't perform post clusterup and also the application and config stuff done on first time itself.